tox.ini

```diff
@@ -148,20 +165,10 @@ commands =
 [testenv:apidocs]
 deps=https://github.com/twisted/pydoctor/archive/3f9c64829dfa040b334c9ae27c332c7078356e79.zip
 
-[testenv:manifest-checker]
```

Comment:
OK, looks like this block which got deleted moved to: https://github.com/twisted/twisted/pull/1380/files#diff-b91f3d5bd63fcd17221b267e851608e8R108